packages/core-data/src/entities.js

```diff
@@ -217,6 +217,7 @@ async function loadPostTypeEntities() {
 			baseURLParams: { context: 'edit' },
 			name,
 			label: postType.labels.singular_name,
+			labelPlural: postType.labels.name,
```

Comment:
the plural form is already available in `postType.name`, I don't mind having it here but I think the main problem right now is that the singular label is not available in the request using the "view" context. (The plural is available in all contexts)

Reply:
Yeh agree. That's why I don't think we could merge a PR like this until the Types API returns more label information for `view` context requests.

Honestly I think it should be possible and feasible given that labels are not a sensitive piece of data.

Reply:
@spacedmonkey Do you see it as achievable that we could get the Types API endpoint to return full `labels` information about the Post Type for both `view` and `edit` contexts?